src/aggregate/aggregate_plan.h

```diff
@@ -96,6 +96,7 @@ typedef struct {
   int isLimited;                  // Flag if `LIMIT` keyword was used.
   uint64_t offset;                // Seek results. If 0, then no paging is applied
   uint64_t limit;                 // Number of rows to output
+  bool runLocal;                  // Indicator that this step should run only local (not in shards)
```

Comment:
Nitpicking: consider putting this after `isLimited`, and changing it to `bool` as well to save on the struct size